media/doc/README.WINE

```diff
@@ -264,7 +264,8 @@ advapi32 -
   reactos/dll/win32/advapi32/wine/security.c     # Out of Sync
 
 gdi32 -
-  reactos/dll/win32/gdi32/objects/linedda.c     # Synced at 20090410
+  reactos/dll/win32ss/gdi32/objects/linedda.c     # Synced at 20090410
```

Comment:
What has changed there?

Reply:
I updated the file location while adding the bidi line *shrougs*